spec/bookingsync/api/client/messages_spec.rb

```diff
@@ -74,4 +74,18 @@
       end
     end
   end
+
+  describe ".add_attachment_to_message", :vcr do
+    let(:attachment_to_be_added) { BookingSync::API::Resource.new(client, id: 1) }
+    let(:prefetched_message_id) {
+      find_resource("#{@casette_base_path}_messages/returns_messages.yml", "messages")[:id]
+    }
+    let(:attributes) { { id: attachment_to_be_added.id } }
+
+    it "adds attachment to given message" do
```

Comment:
how about checking as well returned response? and making sure that `massage.links.attachments` includes attachment id?